dev/CommandBarFlyout/InteractionTests/CommandBarFlyoutTests.cs

```diff
@@ -195,6 +197,269 @@ public void CanTapOnSecondaryItems()
+
+                    Log.Comment("Switch to RightToLeft FlowDirection.");
+                    isRTLCheckBox.Toggle();
+                    Wait.ForIdle();
+                }
+
+                string rightStr = inRTL ? "Left" : "Right";
+                string leftStr = inRTL ? "Right" : "Left";
+                Key rightKey = inRTL ? Key.Left : Key.Right;
+                Key leftKey = inRTL ? Key.Right : Key.Left;
+
+                Log.Comment("Tap on a button to show the CommandBarFlyout.");
+                InputHelper.Tap(showCommandBarFlyoutButton);
+
+                Log.Comment("Press " + rightStr + " key to move focus to second primary command: Copy.");
+                KeyboardHelper.PressKey(rightKey);
+                Wait.ForIdle();
+
+                Button copyButton1 = FindElement.ById<Button>("CopyButton1");
+                var copyButtonElement = AutomationElement.FocusedElement;
+                Verify.AreEqual(copyButtonElement.Current.AutomationId, copyButton1.AutomationId);
+
+                Log.Comment("Press " + leftStr + " key to move focus back to first primary command: Cut.");
+                KeyboardHelper.PressKey(leftKey);
+                Wait.ForIdle();
+
+                Button cutButton1 = FindElement.ById<Button>("CutButton1");
+                var cutButtonElement = AutomationElement.FocusedElement;
+                Verify.AreEqual(cutButtonElement.Current.AutomationId, cutButton1.AutomationId);
+
+                Log.Comment("Press " + leftStr + " key and remain on first primary command: Cut.");
+                KeyboardHelper.PressKey(leftKey);
+                Wait.ForIdle();
+
+                cutButtonElement = AutomationElement.FocusedElement;
+                Verify.AreEqual(cutButtonElement.Current.AutomationId, cutButton1.AutomationId);
+
+                Log.Comment("Press " + rightStr + " key to move focus to MoreButton.");
+                for (int i = 0; i <= 6; i++)
+                {
+                    KeyboardHelper.PressKey(rightKey);
+                    Wait.ForIdle();
+                }
+
+                Button moreButton = FindElement.ById<Button>("MoreButton");
+                var moreButtonElement = AutomationElement.FocusedElement;
+                Verify.AreEqual(moreButtonElement.Current.AutomationId, moreButton.AutomationId);
+
+                Log.Comment("Press " + rightStr + " key and remain on MoreButton.");
+                KeyboardHelper.PressKey(rightKey);
+                Wait.ForIdle();
+
+                moreButtonElement = AutomationElement.FocusedElement;
+                Verify.AreEqual(moreButtonElement.Current.AutomationId, moreButton.AutomationId);
+            }
+        }
+
+        [TestMethod]
+        public void VerifyUpAndDownNavigationBetweenPrimaryAndSecondaryCommands()
+        {
+            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone2))
```

Comment:
Didn't you set the behavior you're testing here to only work in RS3+?

Reply:
All the behaviors tested here are working fine with RS3 and RS3+. On RS3 we keep the secondary commands' IsTabStop set to True and that helps.

Reply:
All the behaviors tested here are working fine with RS2 and RS3+. On RS2 we keep the secondary commands' IsTabStop set to True and that helps.